app/controllers/application_controller.rb

```diff
@@ -2,6 +2,10 @@ class ApplicationController < ActionController::API
   skip_before_filter :verify_authenticity_token
   before_filter :add_allow_credentials_headers
 
+  include ActionController::Helpers
```

Comment:
These were all required/suggested to be added by researched I did with making devise behave properly, lmk if you have any deeper insight